### ISSUE TITLE
Create CommentSection component in Frontend

### ISSUE THREAD

**Issue by CahidArda** [bounswe/bounswe2022group7#270]
We need to create a comment section component for the front end. There will be a comment section under art items and exhibitions. Additionally, there will be a list of replies under a discussion forum post. A comment section component needs to be written to handle these use cases.

Following sections shall exist under a comment section:
- Comments made in the past. If the user is who made the comment, the shall be able to edit it. I won't handle the edit yet since we haven't implemented authentation.
- User shall be able to make comments.

Deadline: 20.10.2022, 23:59
Reviewer: @erimerkin

**Comment by CahidArda** [comment]
I have implemented the comment section. You can view it in `http://localhost:1234/art_item/1`:

![image](https://user-images.githubusercontent.com/57228345/196614936-4c57289f-4dda-454a-ba14-4794bcf875cd.png)

There are three components:
- Comment: For visualising each comment
- NewComment: For adding the new comment text field.
- CommentSection: For creating the comment section with the Comments and the NewComment component.

**Comment by CahidArda** [comment]
I have updated the comments section. I added a menu and a like button using the [MUI guide on cards, section 'complex interaction'](https://mui.com/material-ui/react-card/#complex-interaction). It looks like this:

![image](https://user-images.githubusercontent.com/57228345/197254960-32e54fff-829d-4939-a27e-762875b6572a.png)

Nothing happens yet when you click the menu or the like button. I have seen how one can make configure these components in the [MUI guide on menus](https://mui.com/material-ui/react-menu/). We can use it later.

Ready for your review @erimerkin.
